src/components/watch/WatchToolbar.vue

```diff
@@ -39,61 +39,21 @@
             </v-icon>
           </v-btn>
         </template>
-        <v-list dense>
-          <v-list-item :href="`https://youtu.be/${video.id}`" target="_blank">
-            <v-icon left color="red">
-              {{ icons.mdiYoutube }}
-            </v-icon>
-            {{ $t("views.settings.redirectModeLabel") }}
-          </v-list-item>
-          <v-list-item @click.stop="copyLink">
-            <v-icon left>
-              {{ icons.mdiClipboardPlusOutline }}
-            </v-icon>
-            {{ $t("component.videoCard.copyLink") }}
-          </v-list-item>
-          <v-list-item
-            :disabled="video.type === 'clip'"
-            :to="`/multiview/AAUY${video.id}${getChannelShortname(video.channel)}%2CUAEYchat`"
-          >
-            <v-icon left :color="video.type === 'clip' ? 'grey' : ''">
-              {{ icons.mdiViewDashboard }}
-            </v-icon>
-            {{ $t("component.mainNav.multiview") }}
-          </v-list-item>
-          <v-list-item @click="$store.commit('setReportVideo', video)">
-            <v-icon left :color="video.type === 'clip' ? 'grey' : ''">
-              {{ icons.mdiFlag }}
-            </v-icon>
-            {{ $t("component.reportDialog.title") }}
-          </v-list-item>
-        </v-list>
+        <video-card-menu :video="video" />
```

Comment:
👀 oh we could've done this? lol

Reply:
![image](https://user-images.githubusercontent.com/1288549/128395227-935596d0-535e-4a12-9982-bae6bb431336.png)
Yeah, it looks good
and be consistent with video card